Makefile

```
@@ -27,7 +27,7 @@ DOCKER_IMAGE_DIRS := $(patsubst %/Dockerfile,%,$(DOCKERFILES))
 BUILD_IN_CONTAINER ?= true
 
 # ensure you run `make drone` after changing this
-BUILD_IMAGE_VERSION := 0.20.1
+BUILD_IMAGE_VERSION := 0.20.3
```

Since you recent change just overrode `0.20.2` it should be that
```suggestion
BUILD_IMAGE_VERSION := 0.20.2
```

---

0.20.2 was built but never been used, see https://github.com/grafana/loki/commit/435391661f4e0f03eb378f77c07aaad5009cd6e9#diff-60045f9474705196e786df20a7bb40d29e013e47a3b6927153a10b9acac82266L358